content/blog/2018/07/2018-07-23-remoting-kafka-plugin-1.0-release.adoc

```diff
@@ -69,7 +69,6 @@ image:/images/post-images/remoting-kafka/kafka-commands.png[title="Kafka communi
 
 Remoting operations are being executed over Kafka. In the log you may see:
 
-* Command execution (SlaveInstallerFactoryImpl.isWindows())
```

Comment:
I'd keep this since it is part of the stack trace

Reply:
Okay